job_definitions/functional_tests.yml

```diff
@@ -72,6 +72,8 @@
           export DM_SEARCH_API_DOMAIN="{{ app_urls[job.environment].search_api }}"
           export DM_SEARCH_API_ACCESS_TOKEN="$DM_SEARCH_API_TOKEN_{{ job.environment|upper }}"
 
+          export DM_ANTIVIRUS_API_DOMAIN="{{ app_urls[environment].antivirus_api }}"
```

Comment:
Should this one be `job.environment`?

Reply:
ahyes, it was copied from `smoke_tests.yml` which has its variables set up differently. Nice one.